package.json

```diff
@@ -28,6 +28,7 @@
     "http-proxy": "^1.18.0",
     "humps": "^2.0.1",
     "ip": "^1.1.5",
+    "ip-geolocation-api-javascript-sdk": "^1.0.7",
```

Comment:
The `geoip-lite` dependency is now unused right?  Let's remove it

Reply:
thanks, done